pages/platform/data-processing/01_CONCEPTS_Capabilities/guide.en-asia.md

```diff
@@ -6,7 +6,7 @@ section: Concepts
 order: 1
 ---
 
-**Last updated 23<sup>rd</sup> February, 2022**
+**Last updated 22<sup>rd</sup> August, 2022**
```

Comment:
Hi @pyverdon, just a small typo here. it should be "**Last updated 22<sup>nd</sup> August, 2022**"
nd instead of rd

Reply:
Hi, thanks for spotting it. 
I have fixed in today commit https://github.com/ovh/docs/pull/3375/commits/21e1a6c1d1feb907a6b1119e36d7a009444fb86e